src/Misc/externals.sh

```diff
@@ -137,8 +137,7 @@ if [[ "$PLATFORM" == "windows" ]]; then
     acquireExternalTool "$CONTAINER_URL/vstsom/m103_43d97d9c/vstsom.zip" vstsom
     acquireExternalTool "$NODE_URL/v${NODE_VERSION}/win-x64/node.exe" node/bin
     acquireExternalTool "$NODE_URL/v${NODE_VERSION}/win-x64/node.lib" node/bin
-    acquireExternalTool "$CONTAINER_URL/NuGet/3.5.1-beta1-1621/NuGet.zip" nuget
-    acquireExternalTool "$CONTAINER_URL/CredentialProvider.TeamBuild/15.104.25518.0/CredentialProvider.TeamBuild.zip" nuget
+    acquireExternalTool "https://dist.nuget.org/win-x86-commandline/v3.4.3/nuget.exe" nuget
```

Comment:
This should only be going to the dev15rc1 branch, so unless someone randomly picks up this "outdated" release, it should only affect on prem.